src/types/companies.ts

```diff
@@ -0,0 +1,9 @@
+export interface Companies {
+  [name: string]: number;
+}
+
+export interface Company {
+  id: string;
```

Comment:
This looks like a typo.

Was it intentional to declare the type of `id` as `string` (i.e. not a `number`)? In the JSON returned from the `/company` REST endpoint company ids are numbers.